### PR TITLE
Book preview page (Manager)

### PR DESCRIPTION
**Pull request by lexussuxel**
Design book preview according to Figma:
image
image

### CODE REVIEW

File: packages/frontapp/src/app/components/BookInfo/index.tsx [hunk 1]
```diff
@@ -3,10 +3,16 @@ import bookImage from '../../../assets/MOC-data/BookImage.png';
 import React, { FC, useCallback, useEffect, useState } from 'react';
 import { colors, dimensions } from '@mimir/ui-kit';
 import { ReactComponent as Claim } from '../../../assets/claim.svg';
+import { ReactComponent as Edit } from '../../../assets/Edit.svg';
```
Comment (x8er) [inline packages/frontapp/src/app/components/BookInfo/index.tsx:6]: You need to replace the icon with the one in Figma. I already talked to the designer about some export issues, but I couldn't reproduce them. Looking at the svg code itself, it also looks normal

**Review by x8er — CHANGES_REQUESTED** [review]

File: packages/frontapp/src/app/components/BookInfo/index.tsx (hunk 1 above)
Comment (lexussuxel) [reply packages/frontapp/src/app/components/BookInfo/index.tsx:6]: I've changed it

File: packages/apiserver/src/ormconfig.ts [hunk 2]
```diff
@@ -1,12 +1,5 @@
 import { ConnectionOptions } from 'typeorm';
 
-import { Location } from './resources/locations/location.entity';
-import { Material } from './resources/materials/material.entity';
-import { Person } from './resources/persons/person.entity';
-import { Message } from './resources/messages/messages.entity';
-import { Notification } from './resources/notifications/notification.entity';
-import { Status } from './resources/statuses/status.entity';
```
Comment (x8er) [inline packages/apiserver/src/ormconfig.ts]: You need to leave it

**Review by x8er — CHANGES_REQUESTED** [review]
You need to leave it

**Review by sabrina34352 — APPROVED** [review]

**Pull request merged**